src/wren/Scene.cpp

```diff
@@ -632,10 +632,20 @@ namespace wren {
       const float distance = glm::distance(boundingSphere.mCenter, positionalLight->position());
       const float radius = positionalLight->radius() + boundingSphere.mRadius;
       // Check if light is too far away
-      visible = (distance <= radius &&
-                 positionalLight->attenuationConstant() +
-                     distance * (positionalLight->attenuationLinear() + distance * positionalLight->attenuationQuadratic()) <
-                   255.0f);
+      visible = (distance <= radius && (distance - boundingSphere.mRadius <
+                                          0 ||  // Light is inside the boundingSphere, necessary because pow(distance -
```

Comment:
That seems simpler and clearer to me:
```suggestion
      visible = (distance <= radius && (distance < boundingSphere.mRadius
                                          ||  // Light is inside the boundingSphere, necessary because pow(distance -
```